packages/local_auth/local_auth_darwin/darwin/Classes/FLALocalAuthPlugin.m

```diff
@@ -153,6 +166,19 @@ - (void)showAlertWithMessage:(NSString *)message
           dismissButtonTitle:(NSString *)dismissButtonTitle
      openSettingsButtonTitle:(NSString *)openSettingsButtonTitle
                   completion:(FLADAuthCompletion)completion {
+#if TARGET_OS_OSX
+  NSAlert *alert = [[NSAlert alloc] init];
+  [alert setMessageText:message];
+  [alert addButtonWithTitle:dismissButtonTitle];
+  NSWindow *window = self.registrar.view.window;
+  [alert beginSheetModalForWindow:window
+                completionHandler:^(NSModalResponse returnCode) {
+                  [self handleSucceeded:NO withCompletion:completion];
+                }];
+  return;
+#endif
+
+#if TARGET_OS_IOS
```

Comment:
These are mutually exclusive blocks, so should use an `#if/#else` construction, not two separate `#if`s.